### ISSUE TITLE
Add missing notification properties

### ISSUE THREAD

**Issue by bgaillard** [gomoob/php-pushwoosh#31]
It seems Pushwoosh added new Android notification properties in their documentation, we should reflect this in our implementation. 

```
// Android related
"android_root_params": {"key": "value"}, // custom key-value object. root level parameters for the android payload recipients
"android_sound" : "soundfile", // Optional. No file extension
"android_header":"header",    // Optional. Android notification header
"android_icon": "icon",
"android_custom_icon": "http://example.com/image.png", // Optional. Full path URL to the image file
"android_banner": "http://example.com/banner.png", // Optional. Full path URL to the image file
"android_badges": 5, // Optional, integer. Android application icon badge number
"android_gcm_ttl": 3600, // Optional. Time to live parameter - the maximum lifespan of a message in seconds
 
"android_vibration": 0,   // Android force-vibration for high-priority pushes, boolean
"android_led":"#rrggbb",  // LED hex color, device will do its best approximation
"android_priority":-1,  // Priority of the push in the Android push drawer, valid values are -2, -1, 0, 1 and 2
"android_ibc":"#RRGGBB",  // Icon background color on Lollipop, #RRGGBB, #AARRGGBB, "red", "black", "yellow", etc.
```

We should also check properties for other OS.

**Comment by curtisgibby** [comment]
:+1: I'm looking forward to using the `android_ibc` (background color) functionality

**Comment by bgaillard** [comment]
Ok this is now available in version [1.2.0](https://github.com/gomoob/php-pushwoosh/releases/tag/1.2.0).
